nano/node/active_transactions.hpp

```diff
@@ -44,6 +44,23 @@ class election_status final
 	std::chrono::milliseconds election_duration;
 };
 
+class transaction_counter final
+{
+public:
+	transaction_counter ();
```

Comment:
No need to supply an empty default constructor here

Reply:
thanks, forgot i removed everything from this :)